geometry_msgs/Point size
geometry_msgs/Point start_pos
geometry_msgs/Point end_pos
float32 angle
int8 picking_side
int64 parcel_rotation
